README.md

```diff
@@ -30,7 +30,9 @@ After starting their travels, Master and Alice decided to separate and they had
     );
 ```
 
-![alt text](images/MA_1.png)
+![alt text](images/dark_theme/MA_1.png#gh-dark-mode-only)
+![alt text](images/light_theme/MA_1.png#gh-light-mode-only)
```

Comment:
This has to be adapted in order to selectively show the desired picture based on the theme (like in the [documentation](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to)):

```suggestion
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="images/dark_theme/MA_1.png">
  <source media="(prefers-color-scheme: light)" srcset="(images/light_theme/MA_1.png">
  <img alt="Alternative text." src="(images/light_theme/MA_1.png">
</picture>
```

Currently this is using the legacy way of selecting the image based on theme:

> The old method of specifying images based on the theme, by using a fragment appended to the URL (#gh-dark-mode-only or #gh-light-mode-only), is deprecated and will be removed in favor of the new method described above.

Could you update the other instances and replace the `Alternative text` with a simple summary of what is in the image?